### DISCUSSION TITLE
Does Prisma manage connections well in Next.js serverless API routes?

### DISCUSSION THREAD

**Discussion by a7medev** [prisma/prisma#9668] [Q&A] [answered]
I want to know if Prisma manages connections in a good way when used in a serverless environment like Next.js API routes or does it make a bunch of connection pools for every API request?

If not, can I combine it with `serveless-mysql` or `serverless-postgres` packages?

**Comment by ryands17** [comment, marked as answer]
@a7med-mahmoud 👋 

Yes it should manage it well. Have a look at this [doc](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#recommended-connection-pool-size-1) for more details on how to have an optimal number of connections.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
